libsplinter/src/service/timer_handler_factory.rs

```diff
@@ -12,13 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! Contains `TimerHandlerFactory` trait.
+
 use crate::error::InternalError;
 
 use super::TimerHandler;
 
+/// Used to create new `TimerHandler`s
```

Comment:
Prefer `TimerHandler` instances over `TimerHandler`'s, as it looks strange to pluralize a code block with non-code text. Likewise elsewhere in the PR.

Reply:
fixed